### PR TITLE
01-numpy.md: fix typos, reword, improve

### PR DESCRIPTION
**Pull request by maxim-belkin**
See discussion here: https://github.com/swcarpentry/python-novice-inflammation/pull/538

### CODE REVIEW

**Comment by maxim-belkin** [comment]
@ldko would you be willing to comment on this PR?

File: _episodes/01-numpy.md [hunk 1]
```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
 
 In Python, variable names:
 
- - must begin with a letter, and
+ - variable names can include letters, digits, and underscores, but cannot
```
Comment (ldko) [inline _episodes/01-numpy.md]: The "variable names" doesn't need to be here.

File: _episodes/01-numpy.md [hunk 2]
```diff
@@ -136,7 +137,7 @@ print(weight_kg)
 ~~~
 {: .output}
 
-To change variable's value, we have to assign it a new one:
+To change value of the `weight_kg` variable, we have to assign it a new one:
```
Comment (ldko) [inline _episodes/01-numpy.md]: This could be "To change the value"...

Comment (maxim-belkin) [reply _episodes/01-numpy.md]: I'll disagree here. If we say `To change the value`, the complete sentence would read:
> To change the value, we have to assign it a new one

and `it` in the second half of the sentence becomes ambiguous.

File: _episodes/01-numpy.md [hunk 3]
```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
 
 In Python, variable names:
 
- - must begin with a letter, and
+ - can include letters, digits, and underscores, but cannot
+   start with a digit
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: shall we make an independent bullet point out of`cannot start with a digit`?

Comment (ldko) [reply _episodes/01-numpy.md]: Sounds good to me.

File: _episodes/01-numpy.md (hunk 2 above)
Comment (ldko) [reply _episodes/01-numpy.md]: Ok, leaving it as you have it works for me.

Comment (maxim-belkin) [reply _episodes/01-numpy.md]: how about something like this:
> To change the value of the `weight_kg` variable, we have to **assign** `weight_kg` to a new value using the equals `=` sign.

File: _episodes/01-numpy.md [hunk 4]
```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
 
 In Python, variable names:
 
- - must begin with a letter, and
+ - can include letters, digits, and underscores
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: include -> contain?

File: _episodes/01-numpy.md [hunk 5]
```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
...
+ - cannot start with a digit
  - are [case sensitive]({{ page.root }}/reference/#case-sensitive).
 
 This means that, for example:
  - `weight0` is a valid variable name, whereas `0weight` is not
  - `weight` and `Weight` are different variables
 
 ## Types of data
-Python knows various types of data. The most common ones are:
+Python knows various types of data. Three common ones are:
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: common -> basic?

File: _episodes/01-numpy.md [hunk 6]
```diff
@@ -90,7 +91,7 @@ weight_kg_text = 'weight in kilograms:'
 {: .language-python}
 
 ## Using Variables in Python
-To display the value of a variable to the screen in Python, we can use `print` function:
+To display the value of a variable to the screen in Python, we can use the `print` function:
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: display -> see ? (because we're talking about our selves)

File: _episodes/01-numpy.md (hunk 5 above)
Comment (ldko) [reply _episodes/01-numpy.md]: I am not sure that "basic" is better, but I think either way works.

File: _episodes/01-numpy.md (hunk 2 above)
Comment (ldko) [reply _episodes/01-numpy.md]: I think of assigning a value to a variable, not the other way around, so I suggest a slight change of removing "to":
"To change the value of the weight_kg variable, we have to assign weight_kg a new value using the equals = sign."

**Comment by maxim-belkin** [comment]
Thanks @ldko

**Pull request merged**